runtime/vanilla/src/lib.rs

```diff
@@ -398,15 +392,18 @@ pub type TimeStampedPrice = orml_oracle::TimestampedValue<OraclePrice, Moment>;
 pub struct AggregatedDataProvider;
 impl DataProvider<CurrencyId, TimeStampedPrice> for AggregatedDataProvider {
     fn get(key: &CurrencyId) -> Option<TimeStampedPrice> {
-        VanillaOracle::get(key)
+        if let Some(price) = VanillaOracle::get(key) {
+            log::info!("DataProvider get price {:?}", price);
```

Comment:
I think you can print Option<Price> directly.

Reply:
fixed:  removed the log directly.